test/settings_test.rb

```diff
@@ -89,7 +89,7 @@ def test_multithreaded_create
   end
 
   def test_complex_serialization
-    complex = [1, '2', {:three => true}]
+    complex = [1, '2', {"three" => true}]
```

Comment:
This change makes me nervous, is it going to be a behavior change?  Will people upgrading to newest Rails need to make this change in other parts of their app, and will this work right with older Rails?

Reply:
When I undid line 92's change, I go this:
```
Failure:
SettingsTest#test_complex_serialization [/Users/macbookpro/Projects/setler/test/settings_test.rb:94]:
Expected: [1, "2", {:three=>true}]
  Actual: [1, "2", {"three"=>true}]
```

Reply:
Hmm. Maybe @alindeman knows up-to-date Ruby/Rails and has ideas if this is something I should be worrying about?

Reply:
OK

Reply:
Unfortunately I'm not sure offhand, though I do share the concern that this might break apps. However, I don't know if `setler` can do much about it without making larger, even riskier changes because `setler` depends on the underlying behavior of `serialize` in ActiveRecord.

But I can't commit to spending time looking into it deeply right now. If I did, though, I'd think about creating a small application that used `serialize :foo`, and see if the behavior of setting (and then retrieving) `foo = {:three => true}` changed in Rails versions over time.

If this is a behavior change in Rails, it might be acceptable to document it in the Changelog and move on. A workaround would be for applications to themselves call `symbolize_keys` or `deep_symbolize_keys` when retrieving values. This would also make the desire explicit, which might be a desirable thing anyway.

Reply:
Thanks Andy!  Since we are 'just' using ActiveRecord serialize, it's probably a safe gamble that setler will do the right thing depending on what version of ActiveRecord people are using. I'll go ahead and merge this as is.